### DISCUSSION TITLE
可以支持下纯小鹤音形方案吗，四字上屏的那种

### DISCUSSION THREAD

**Discussion by bestdonger** [Mintimate/oh-my-rime#143] [Ideas]
参考小鹤官方：http://flypy.ysepan.com/
如果能放到薄荷里作为一种方案就太好了

**Comment by Mintimate** [comment]
小鹤双拼的音形是不开源的，引入只能引入第三方。 四字上屏会导致其他已经规划好的反查等功能无法使用，所以暂时不考虑。
